config/jobs/kubernetes/kops/kops-periodics-conformance.yaml

```diff
@@ -229,7 +97,7 @@ periodics:
           -v 2 \
           --up --down \
           --cloud-provider=aws \
-          --create-args="--image='099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-arm64-server-20220901' --channel=alpha --networking=calico --container-runtime=containerd --zones=eu-central-1a --node-size=t4g.large --master-size=t4g.large --discovery-store=s3://k8s-kops-prow/discovery" \
+          --create-args="--image='099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-arm64-server-20220810' --channel=alpha --networking=calico --container-runtime=containerd --zones=eu-central-1a --node-size=t4g.large --master-size=t4g.large --discovery-store=s3://k8s-kops-prow/discovery" \
```

Comment:
this is going back in time

Reply:
wait ill push the merge along with it as thats not the issue.... this one unit test failed again...even though its working fine on local